### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by anton-bondarev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,30 @@
 Embox [![Build Status](https://travis-ci.org/embox/embox.svg?branch=master)](https://travis-ci.org/embox/embox) [![Coverity Scan Build Status](https://scan.coverity.com/projects/700/badge.svg)](https://scan.coverity.com/projects/700)
 =====
 
-Embox is a configurable operating system kernel designed for resource
-constrained and embedded systems.
+Embox is a configurable real-time operating system designed for resource constrained and embedded systems.
+
+Key features
+---------------
+* Support POSIX
```
Comment (AntonKozlov) [inline README.md]: `Implements POSIX subsets`?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+* Support C++
+* Support file system
```
Comment (AntonKozlov) [inline README.md]: List of supported file systems would look very natural here

File: README.md (hunk 1 above)
Comment (Deryugin) [reply README.md]: I think `POSIX-compliant` is fine

File: README.md (hunk 2 above)
Comment (Deryugin) [reply README.md]: Done

**Pull request merged**